authorization/3.0/_index.md

```diff
@@ -31,7 +31,7 @@ The primary goal of this release is to make Jakarta Authorization future proof b
 * [Jakarta Authorization 3.0 Specification Document](./jakarta-authorization-spec-3.0.pdf) (PDF)
 * [Jakarta Authorization 3.0 Specification Document](./jakarta-authorization-spec-3.0.html) (HTML)
 * [Jakarta Authorization 3.0 Javadoc](./apidocs)
-* [Jakarta Authorization 3.0 TCK](https://download.eclipse.org/jakartaee/authorization/3.0/jakarta-authorization-tck-3.0.0.zip)([sig](https://download.eclipse.org/jakartaee/authorization/3.0/jakarta-authorization-tck-3.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/authorization/3.0/jakarta-authorization-tck-3.0.0.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
+* [Jakarta Authorization 3.0 TCK](https://download.eclipse.org/jakartaee/authorization/3.0/jakarta-authorization-tck-3.0.0.zip) ([sig](https://download.eclipse.org/jakartaee/authorization/3.0/jakarta-authorization-tck-3.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/authorization/3.0/jakarta-authorization-tck-3.0.0.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
```

Comment:
What changes are you doing here? I feel it is identical.

Reply:
It's adding a space between `Jakarta Authorization 3.0 TCK` and `(sig,sha,pub)`, so it doesn't get rendered as `Jakarta Authorization 3.0 TCK(sig,sha,pub)`.

Reply:
ah, thanks. Did not spot the space :)